modules/web-console/frontend/app/components/bs-select-menu/template.pug

```diff
@@ -37,11 +37,11 @@ ul.bs-select-menu(
             ng-click='$select($index, $event); $event.stopPropagation();'
             ng-class=`{ 'bssm-item-button__active': $isActive($index) }`
             data-placement='right auto'
-            title='{{ ::match.label | bsSelectStrip }}'
+            title='{{ match.title || match.label | bsSelectStrip }}'
         )
             img.bssm-active-indicator.icon-left(
                 ng-src='{{ $isActive($index) ? "/images/checkbox-active.svg" : "/images/checkbox.svg" }}'
             )
-            span.bssm-item-text(ng-bind-html='match.label')
+            .bssm-item-text(ng-bind-html='$ctrl.trustHtml(match.label)')
```

Comment:
Why `trustHtml` in menu template if you can do the same when building menu in ignite-chart-series-selector?